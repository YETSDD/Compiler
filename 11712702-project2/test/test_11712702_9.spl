int test_11712702_9(int a[2], int b)
{
  char c = 'c';
  if (a[0] > b)
  {
    return a[c];
  }
  else
  {
    return b;
  }
}
