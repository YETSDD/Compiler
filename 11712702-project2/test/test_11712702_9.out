Error type 12 at Line 6: array indexing with non-integer value
Error type 8 at Line 6: incompatiable return type
